### DISCUSSION TITLE
Can I change the standard names A and B with the implicit "many to many" relation?

### DISCUSSION THREAD

**Discussion by Kicshikxo** [prisma/prisma#17260] [Q&A] [answered]
I use the standard implicit many-to-many model like:
```
model Post {
  id         Int        @id @default(autoincrement())
  title      String
  categories Category[] @relation(name: "category_to_post")

  @@map("posts")
}

model Category {
  id    Int    @id @default(autoincrement())
  name  String
  posts Post[] @relation(name: "category_to_post")

  @@map('categories')
}
```
and got SQL code like this:
```sql
CREATE TABLE "categories" (
    id SERIAL PRIMARY KEY,
    name TEXT NOT NULL
);

CREATE TABLE "posts" (
    id SERIAL PRIMARY KEY,
    title TEXT NOT NULL
);

CREATE TABLE "_category_to_post" (
    "A" integer NOT NULL REFERENCES "categories"(id),
    "B" integer NOT NULL REFERENCES "posts"(id)
);

CREATE UNIQUE INDEX "_category_to_post_AB_unique" ON "_category_to_post"("A" int4_ops,"B" int4_ops);
CREATE INDEX "_category_to_post_B_index" ON "_category_to_post"("B" int4_ops);
```
I can rename the relationship table in ```@relation```, but can I rename the A and B fields in that table without creating the table explicitly?

**Comment by nurul3101** [comment, marked as answer]
Hello @Kicshikxo 👋 

At the moment, it's not possible to rename the column names.
We have a feature request for it here:
- https://github.com/prisma/prisma/issues/10243

If you can leave a 👍 to the feature request then it would help our product team in prioritising it.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
